### ISSUE TITLE
Date header sometimes repeats

### ISSUE THREAD

**Issue by dgw** [marumari/textual-theme-Equinox#23]
The following log (some details obfuscated) happened tonight:

````
September 18, 2016
[00:03:19] -owner- owner is inviting newguy to #channel.
September 18, 2016
[01:19:05] <+dgw> newguy where
````

**Comment by marumari** [comment]
Yeah, I've been seeing that as well.  Somebody, I will be annoyed enough to go in and fix it.  :)

**Comment by dgw** [comment]
Just observed it this afternoon in a PM, between privmsgs without anything else. Granted, the first message in the buffer came from ZNC playback, with a date header inserted before it, then another date header was inserted before my response to it. But still odd.

````
[Textual loaded query history, last message included for context]
[15:46:42] <dgw> mm
October 8, 2016
[08:03:20] <other_person> [message redacted]
October 8, 2016
[16:08:20] <dgw> or not
````

**Comment by marumari** [comment]
BTW, if you can ever get me the actual DOM of these lines that would be super helpful to debug these.  :)

**Comment by emsquared** [comment]
Received an e-mail with similar issue.

```
Bug. Occurred when:
- Left Macbook on power, lid closed, textual still running, connected to USB-C Dock (has monitor on it on DisplayPort)
- Come back after a while, open lid, then unplug Dock cable
- Macbook resets to single monitor mode (internal display) 
- Textual looks like this, and it keeps doing this… For every new message, it keeps doing this. Have to restart Textual to fix.
Note that it only does that in the channel that’s currently open. All other channels are fine.
```
<img width="1021" alt="schermafbeelding 2017-04-09 om 01 54 25" src="https://cloud.githubusercontent.com/assets/341587/24875285/9b9583b8-1df5-11e7-997f-725073fd9dbf.png">

**Comment by dgw** [comment]
That looks like a much more severe issue than what I observed. In my case, the header only repeated once or twice at most.

**Comment by sneakyness** [comment]
![screen shot 2017-06-23 at 12 03 32 am](https://user-images.githubusercontent.com/190671/27755655-050c47e2-5dbf-11e7-8383-a49fc58cfe54.png)

Just throwing mine on the pile

**Comment by april** [comment]
Woah that's very strange. Are you able to get me the DOM from when that happens?

**Comment by sneakyness** [comment]
Yeah I'm totally down just tell me how/what you want me to do

**Comment by Necrosan** [comment]
Here is the log you requested of this happening, April.
I put it in ZIP compression for your convenience (and because Github told me I can't upload raw html)
Hopefully this is the last bit of information needed to finally eviscerate this terrible bug. 👍

[equinox_bullcrap.zip](https://github.com/april/textual-theme-Equinox/files/1627225/equinox_bullcrap.zip)
